mscore/uploadscoredialog.cpp

```diff
@@ -69,6 +69,7 @@ UploadScoreDialog::UploadScoreDialog(LoginManager* loginManager)
                            .arg("</a>"));
       QFont font = licenseHelp->font();
       font.setPointSize(8);
+      font.setItalic(true);
       licenseHelp->setFont(font);
 
```

Comment:
Not unsetting italic again? Same font is used later again, and there it was not italic before this change.